app/app.cpp

```diff
@@ -3,5 +3,83 @@
 
 namespace vsite::oop::v3
 {
+	results::results(const int length) {
+		this->length = length;
+		results_array = new student[length];
+	}
 
+	void results::add(const student& student) {
+		this->results_array[counter++] = student;
+	}
+
+	unsigned int results::has_grade(const int grade) const {
+		int num_grade = 0;
```

Comment:
Zašto nije `unsigned int` (ili `uint32_t`) kad već to vraćate iz funkcije? 
Isto vrijedi za `num_letter` funkciju niže.